src/xdocs/config_whitespace.xml

```diff
@@ -388,6 +397,14 @@ class Foo
               }
           }
       </source>
+      <p>
+        An example how to allow no empty line between import and static import:
```

Comment:
"to allow" is not good wording. we have violations or no violations.
Please add code example, users like them the most.